brave/brave-core
# Adds new disabled state to wallet panel

Related issue: https://github.com/brave/brave-browser/issues/3597
UI PR: https://github.com/brave/brave-ui/pull/424

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
1. Launch Brave with a fresh profile
2. Open a new tab, click the Rewards icon in the nav bar
3. Confirm the prompt to opt-in rewards comes up, then opt-in
4. Go to `brave://rewards`, disable Rewards
5. Open a new tab, click the Rewards icon
6. Confirm the new disabled state panel shows up
7. Confirm all links direct properly, and that info in panel is displayed correctly
8. Re-enable Rewards and confirm publisher info is shown for a tab as expected

## Reviewer Checklist:
